GOVERNANCE.MD

```diff
@@ -1,6 +1,92 @@
 # Community Committee Governance
 
-The Community Committee governance is currently under pull request as the Community Committee Charter.
+Section 1. Guiding Principle.
+
+The Community Committee will operate transparently, openly, collaboratively, and ethically. Project proposals, timelines, and status must not merely be open, but also easily visible to the entire ecosystem.
+
+Section 2. Evolution of Node.js Foundation Governance.
```

Comment:
You are correct that corp structures defer to the experts they have put in place. The Board would need a pretty good reason to override their experts! (it _can_ have legal ramifications)

SO, as an example- if someone (or a group of someones) under the TSC's scope disagree with the CoC- for example- they should be encouraged to get involved with the organization that has the authority over it... but no group is allowed to impede or editorialize in an area they have not been empowered to do so by the Board.

This is one of the reasons the TSC just finalized a [detailed Scope of Responsibilities](https://github.com/nodejs/TSC#tsc-scope) that draws as-clear-as-possible lines in the TSC's authority. HTH!